pyccel/commands/console.py

```diff
@@ -234,6 +234,10 @@ def pyccel(files=None, openmp=None, openacc=None, output_dir=None, compiler=None
 
     base_dirpath = os.getcwd()
 
+    if args.language == 'python' and args.output == '':
+        print("Cannot output python file to same folder as this would overwrite the original file. Please specify --output")
+        sys.exit(1)
```

Comment:
When we use `epyccel`, we should also check that the output directory is not `__epyccel__` because we would overwrite the file there. Or is that exactly what you want?

Reply:
I didn't think it would matter if we overwrote that file given that it is a temporary created by pyccel anyway. If we want to save it in a different folder then we need to add some logic to `epyccel.py` to choose the import folder. Currently we do:
```
# Import shared library
sys.path.insert(0, epyccel_dirpath)
```
which means that we expect the importable object to be in the `__epyccel__` folder

Reply:
The logic that needs adding is not particularly complicated though, so I can add it if you think it would be useful to have all 3 python files?

Reply:
I don't think `epyccel` is an issue because it's not created by the user

Reply:
OK then, let's keep things as they are. Thanks!